clang/lib/Sema/SemaDecl.cpp

```diff
@@ -19213,6 +19213,29 @@ void Sema::ActOnFields(Scope *S, SourceLocation RecLoc, Decl *EnclosingDecl,
 
     if (Record && FD->getType().isVolatileQualified())
       Record->setHasVolatileMember(true);
+    auto IsNonDependentBitField = [](const FieldDecl *FD) {
+      if (!FD->isBitField())
```

Comment:
Is this better as `return FD->isBitField() && !FD->getType()->isDependentType();`?